src/app/loadout-builder/process-worker/set-tracker.ts

```diff
@@ -36,13 +36,15 @@ export class SetTracker {
   /**
    * Insert this set into the tracker. If the tracker is at capacity this set or another one may be dropped.
    */
+  // TODO: rewrite this to just use comparators!
```

Comment:
If we do this we need to keep an eye on perf, from memory I didn't do this as we were trying to minimise function calls. This might not be an issue anymore now that this is in a web worker though.

Reply:
Yeah I'll be checking it out. I think I have an idea for how it could go.